lib/module.js

```diff
@@ -195,6 +195,8 @@ Module._findPath = function(request, paths, isMain) {
         if (exts === undefined)
           exts = Object.keys(Module._extensions);
         filename = tryPackage(basePath, exts, isMain);
+      } else if (rc === 2) { // Pipe or Socket
```

Comment:
Removed on new version as suitable comment on that version was something like neither file or directory which can be implied from the above.